sormas-api/src/main/resources/strings_en-AF.properties

```diff
@@ -1148,4 +1148,4 @@ DefaultPassword.ownUserNewPasswordSetHints=Your account has been successfully up
 DefaultPassword.otherUsersNewPasswordSetHints=Your accounts have been successfully updated. Make sure to copy the new passwords and to notify the affected users about the password change. The new passwords are displayed only once.
 DefaultPassword.newPassword=New Password
 DefaultPassword.newPasswordPlaceholder=New Password\: %s
-DefaultPassword.unchanged=- unchanged -
\ No newline at end of file
+DefaultPassword.unchanged=- unchanged -
```

Comment:
As discussed earlier, this should also be removed from all but the strings.properties. Please always check the diff view (the files changed) yourself, before requesting a review!